cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -59,20 +59,19 @@ if [ -n "${SECRET_ARN}" ]; then
   fi
 
   # Decode munge key and write to munge.key file
-  decoded_key=$(printf "%s" "${encoded_key}" | base64 -d)
```

Comment:
Please, leave this line (without using the stored value) so that we are able to capture any failure in the decoding of the fetched encoded key (also because otherwise the following if condition makes no sense).

Reply:
Great point! Thanks for your reminding Jacopo!